makeflow/src/dag_file.c

```diff
@@ -84,4 +87,32 @@ int dag_file_in_trans( const struct dag_file *f )
 		return 0;
 }
 
+int dag_file_mount_clean(struct dag *d) {
```

Comment:
If you can guarantee there is no other places freeing the other fields of `dag_file`, I can enlarge my function to also free other fields.